doc/source/serve/index.md

```diff
@@ -121,9 +126,89 @@ Because it's built on top of Ray, you can run it anywhere Ray can: on your lapto
 
 :::
 
+
+## How can Serve help me as a...
+
+:::{dropdown} Data scientist
+:animate: fade-in-slide-down
+
+Serve is easy to use. You can test your models (and your entire deployment graph) on your local machine before deploying it to production on a cluster. You don't need to know heavyweight Kubernetes concepts or cloud providers to use Serve.
+
+:::
+
+:::{dropdown} ML engineer
+:animate: fade-in-slide-down
+
+Serve helps you scale out the deployment and run them reliably in an efficient, cost-saving manner. Serve offers a first-class model composition API to help you combine models together to build end-to-end application. Additionally, Serve natively run on Kubernetes which minimal operation overhead.
```

Comment:
Serve helps you scale out the deployment and runs them reliably, efficiently, and cheaply to save costs. With Serve's first-class model composition API, you can combine models together and build end-to-end application. Additionally, Serve natively runs on Kubernetes with minimal operation overhead.